opm/models/blackoil/blackoilsolventmodules.hh

```diff
@@ -596,23 +617,44 @@ public:
     void solventPostSatFuncUpdate_(const ElementContext& elemCtx,
                                    unsigned dofIdx,
                                    unsigned timeIdx)
+    {
+        const PrimaryVariables& priVars = elemCtx.primaryVars(dofIdx, timeIdx);
+        this->solventPostSatFuncUpdate_(elemCtx.problem(), priVars, elemCtx.globalSpaceIndex(dofIdx, timeIdx), timeIdx, elemCtx.linearizationType());
+    }
+
+    template <class Problem>
+    struct ProblemAndCellIndexOnlyContext
```

Comment:
Why does this need to be in the class's public interface?

Reply:
It does not, will fix.